### PR TITLE
made full content

### PR DESCRIPTION
**Pull request by vitaliipasiuta**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -14,7 +14,7 @@ body {
 .wrap{
     width: 1090px;
     border: solid 1px #dcdcdc;
-    height: 100%;
+    height: 5000px;
```
Comment (vitaliipasiuta) [inline style.css]: Are you sure, that 5000px is a good idea?

File: style.css [hunk 2]
```diff
@@ -762,6 +762,65 @@ body {
...
+    text-decoration: none;
+    color: #333;
+    margin-left: 40px;
+    display: block;
+}
+.column-right a:hover{
+    text-decoration: underline;
+}
+.rightnews-list2{
+    margin: -10px -40px;
```
Comment (vitaliipasiuta) [inline style.css:798]: Negative margin? Is that possible to get rid of negative values?

**Comment by vitaliipasiuta** [comment]
Looks great, its time to update your CV.
But, please, do smth with this background ;)

File: style.css (hunk 1 above)
Comment (andriihavryliak) [reply style.css]: when I set a percent value, background also get lost

**Pull request merged**